libs/coin-modules/coin-aptos/src/config.ts

```diff
@@ -0,0 +1,27 @@
+import buildConConfig, { type CurrencyConfig } from "@ledgerhq/coin-framework/config";
+
+export type AptosConfig = {
```

Comment:
This is for Alpaca adaptation, you don't need it

Reply:
removed and updated `families/aptos/setup.ts`